packages/@tests/marine/web/test/test.spec.ts

```diff
@@ -0,0 +1,86 @@
+import Webpack from 'webpack';
+import WebpackDevServer from 'webpack-dev-server';
+import webpackConfig from '../webpack.config.js';
+import process from 'process';
+import path from 'path';
+import fs from 'fs';
+
+// change directory to the location to the test-project.
+// run all the subsequent Webpack scripts in that directory
+process.chdir(path.join(__dirname, '..'));
+
+let server;
+
+jest.setTimeout(10000);
+
+const startServer = async (modifyConfig?) => {
+    const loadInBrowserToDebug = false;
+    // const loadInBrowserToDebug = true; // use this line to debug
+
+    modifyConfig = modifyConfig || ((_) => {});
+
+    const cfg: any = webpackConfig();
+    modifyConfig(cfg);
+    const compiler = Webpack(cfg);
+    const devServerOptions = { ...cfg.devServer, open: loadInBrowserToDebug };
+    server = new WebpackDevServer(devServerOptions, compiler);
+    await server.start();
+    // wait for webpack to load
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
there must be a better way. It's just not cool. What happens if you don't do this? If there is some error maybe instead of timeout it's better to have an interval that is constantly checking if there is an error and when it dissapears - resolve the promise. But I hope there is some property or something that can help make sure it's loaded

Reply:
fixed